onboarding/general-member.md

```diff
@@ -0,0 +1,68 @@
+# General Member onboarding
+
+Welcome to the TODO Group!
+
+This guide should provide your company representatives helpful onboarding
+resources once your [TODO General Membership Application](https://joinnow.todogroup.org/)
+has been validated and processed.
+
+An issue should have been created to process your company's membership (via this [template](/.github/ISSUE_TEMPLATE/onboarding-general-member.md)).
+
+If you run into any issues during the onboarding process, please reach out to the TODO Group Program Manager ([@anajsana][todo-pm]).
+
+- [Communication channels](#communication-channels)
+  - [Registration required](#registration-required)
+  - [Social media](#social-media)
+  - [GitHub](#github)
+  - [Private channels](#private-channels)
+- [Meetings and Calls](#meetings-and-calls)
+  - [Public calls](#public-calls)
+  - [Private calls](#private-calls)
+- [Resources and initiatives](#resources-and-initiatives)
+
+## Communication channels
+
+<!--
+TODO: Publish this info on the website to de-duplicate content
+-->
+
+### Registration required
+
+- [TODO Group Slack](https://slack.todogroup.org/)
+- [Community Mailing List](https://docs.google.com/forms/d/e/1FAIpQLSeU0YGM_IJ6gY8E5IIiwXKD_FZi3kAVc4E9_-3dtTDyKMSjdA/viewform)
+- [OSPO Newsletter](https://www.getrevue.co/profile/osponews)
+- [OSPOlogy](https://github.com/todogroup/ospology/discussions) (OSPO Forum Discussions)
```

Comment:
I'll call it "OSPO Forum" and people don't have to register to have access to the content (however, if they want to comment on a discussion, they might need to sign in in GitHub)